pom.xml

```diff
@@ -512,11 +516,12 @@
                 <artifactId>license-maven-plugin</artifactId>
                 <version>${license-maven-plugin.version}</version>
                 <configuration>
-                    <header>com/mycila/maven/plugin/license/templates/MIT.txt</header>
+                    <header>${maven.multiModuleProjectDirectory}/LICENSE.md</header>
```

Comment:
Fetching LICESNE.md is going to create problems and will not align us with the format that we are following.

I really do not encourage this.

Reply:
Are you suggesting I create a LICENSE.txt and reference that file instead of LICENSE.md?

Reply:
@ohbus can you elaborate what's the problem and maybe give some guidance how to solve it?

Reply:
Extremely Sorry for the delayed response. Somehow I missed this notification.


I was actually hoping that this change would effectively change all the license headers that we already have installed in the files and will make us lose the modularity that we have for using the license plugin to help include licenses inside the project files.

Reply:
What you describe is what it does, I am not sure what the concern is?

Reply:
There will be problems like, when generating a license header for the files in our repository, it will include the zkoss header module which we do not require for every file header.

Solution:
- We should continue using the template provided by the plugin's classpath.
- Configure this plugin to skip the zkoss module or to add the necessary GPL header to it.

Reply:
@ohbus  @iluwatar 

Ok, so I guess I misunderstood the initial requirement, I was of the understanding that we need all the headers updated with the inclusion of zkoss related GPL license.

So, I will revert all these changes and instead just update the files within the MVVM pattern to have additional header which would include existing license and zkoss gpl license.

Please reply if this is what we want??

Reply:
Yes please, make sure you amend changes on this PR only and not make any duplicates.

Thanks.

Reply:
done.